packages/@lwc/compiler/src/transformers/javascript.ts

```diff
@@ -7,6 +7,7 @@
 import * as babel from '@babel/core';
 
 import babelClassPropertiesPlugin from '@babel/plugin-proposal-class-properties';
+import babelObjectRestSpreadPlugin from '@babel/plugin-proposal-object-rest-spread';
```

Comment:
It's weird that there's no package.json change, but I checked and this is included because we have `@babel/preset-env`.